packages/syft/src/syft/service/user/user_stash.py

```diff
@@ -76,6 +76,13 @@ def get_by_email(
         qks = QueryKeys(qks=[EmailPartitionKey.with_obj(email)])
         return self.query_one(credentials=credentials, qks=qks)
 
+    def email_exists(self, email="") -> bool:
```

Comment:
Can we skip setting email to empty string. and add missing annotation for email field.